src/ipfs.io/docs/index.html

```diff
@@ -21,22 +21,23 @@
 
 These are links to some of the IPFS repositories.
 
-- [github.com/ipfs/ipfs](https://github.com/ipfs/ipfs) - original IPFS paper. Soon the spec too.
-- [github.com/ipfs/go-ipfs](https://github.com/ipfs/go-ipfs) - the main, reference implementation
-- [github.com/protocol/ipfs-webui](https://github.com/protocol/ipfs-webui) - our web console
-- [Search github for "ipfs"](https://github.com/search?q=ipfs&ref=searchresults&type=Repositories&utf8=%E2%9C%93)
+- [github.com/ipfs/ipfs](https://github.com/ipfs/ipfs) - The original IPFS paper, with the main spec.
+- [github.com/ipfs/go-ipfs](https://github.com/ipfs/go-ipfs) - The main, reference implementation.
+- [github.com/ipfs/specs](https://github.com/ipfs/specs) - Highly technical specs for IPFS.
+- [github.com/protocol/ipfs-webui](https://github.com/protocol/ipfs-webui) - Our web console.
```

Comment:
I dont think it makes sense to direct users here. it's available for them to see with just http://localhost:5001/webui -- a link to the org with all the repos, or to a repo map, would be best.